corehq/apps/translations/templates/app_translations.html

```diff
@@ -52,6 +53,32 @@
                     {% crispy backup_form %}
                 </form>
             </div>
+            <div id="blacklist-trans-form" class="tab-pane">
+                <table class="table table-striped table-hover">
+                    <thead>
+                        <th class="col-sm-2">App</th>
+                        <th class="col-sm-2">Module</th>
+                        <th class="col-sm-2">Detail or List</th>
+                        <th class="col-sm-2">Field Name</th>
+                        <th class="col-sm-4">Default Display Text</th>
+                    </thead>
+                    <tbody>
+                    {% for translation in blacklisted_translations %}
+                        <tr>
+                            <td>{{ translation.app_name }}</td>
+                            <td>{{ translation.module_id }}</td>
```

Comment:
Can we make this a link to the module? So that one can see what module this is coz the name is not present here.

Reply:
Can also reconsider this if decide to put this out as a separate page.